lib/zstd.h

```diff
@@ -80,15 +80,53 @@ ZSTDLIB_API size_t ZSTD_compress( void* dst, size_t dstCapacity,
                                   int compressionLevel);
 
 /*! ZSTD_decompress() :
-    `compressedSize` : must be the _exact_ size of a single compressed frame.
+    `compressedSize` : must be the _exact_ size of some number of compressed and/or skippable frames.
     `dstCapacity` is an upper bound of originalSize.
     If user cannot imply a maximum upper bound, it's better to use streaming mode to decompress data.
     @return : the number of bytes decompressed into `dst` (<= `dstCapacity`),
               or an errorCode if it fails (which can be tested using ZSTD_isError()). */
 ZSTDLIB_API size_t ZSTD_decompress( void* dst, size_t dstCapacity,
                               const void* src, size_t compressedSize);
 
+#define ZSTD_CONTENTSIZE_UNKNOWN (0ULL - 1)
```

Comment:
It's too soon to integrate these new prototypes into `stable`.
They need first to stage through `experimental`.
There is always a chance that we may have missed something, that we'll find out through usages and feedbacks.

So, declare these new definitions behind `ZSTD_STATIC_LINKING_ONLY`